test/celeritas/em/UrbanMsc.test.cc

```diff
@@ -283,7 +291,10 @@ TEST_F(UrbanMscTest, msc_scattering)
         real_type r = i * 2 - real_type(1e-4);
         geo_view    = {{r, r, r}, direction};
 
-        this->set_inc_particle(pdg::electron(), MevEnergy{energy[i]});
+        MevEnergy inc_energy = MevEnergy{energy[i]};
+        this->set_inc_particle(pdg::electron(), inc_energy);
+        PhysicsTrackView phys
+            = this->make_track_view("e-", MaterialId{1}, inc_energy);
```

Comment:
Instead of passing the energy and particle type twice, maybe do `set_inc_particle` inside `make_track_view`? (The `ParticleParams::find` method can take a PDG number so no need to pass a string.)

Reply:
Done.